src/components/20-molecules/cookie-disclaimer/README.md

```diff
@@ -0,0 +1,96 @@
+# Cookie disclaimer
+
+Shows a message that explains the conditions of the Data Protection Act. By
+clicking on the button, the component stores in the local storage the date and time of when the user has clicked
```

Comment:
"the component stores in local state" maybe something like stores a identifier ... . Then the info is missing if the identifier already exist in local storage that the cookie disclaimer don't pop up.

Reply:
done